packages/website/src/theme/NotFound/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.title {
+  font-family: monospace;
+  font-size: 3rem;
+  margin-bottom: 2rem;
+}
+
+.title::before {
+  content: '$ npx eslint .';
```

Comment:
Why is this in CSS not in markup? 🤔

Reply:
No particular reason 😄 screen readers do announce them. I'll just move it to markup.

Reply:
Oh, right, it's that Docusaurus adds default styles to `<code>` and `<pre>` blocks. Switched to a `<div>`.